test/io.jl

```diff
@@ -153,6 +159,47 @@ end
         "<td>16</td><td><div class=\"markdown\"><p><em>A</em>b<strong>A</strong></p>"*
         "\n</div></td></tr></tbody></table>"
 
+    # Test that single and double quotes get escaped properly
+    df = DataFrame(
+        xs = ["'", "\"", "<foo>'</bar>"],
+        ys = [QuoteTestType{'\''}(), QuoteTestType{'"'}, QuoteTestType{Symbol("\"'")}()],
+        zs = QuoteTestType{'"'}[QuoteTestType{'"'}(), QuoteTestType{'"'}(), QuoteTestType{'"'}()],
+    )
+    io = IOBuffer()
+    show(io, "text/html", df)
+    str = String(take!(io))
+    @test str ==
+        "<table class=\"data-frame\"><thead>"*
```

Comment:
can you add a space before `*` everywhere for consistency? Thank you!

Reply:
Done!